Exiled.API/Features/Doors/Gate.cs

```diff
@@ -40,6 +40,21 @@ public Gate(PryableDoor door, List<Room> room)
         /// </summary>
         public IEnumerable<Transform> PryPositions => Base.PryPositions;
 
+        /// <summary>
+        /// Gets a value indicating whether or not the door is fully closed.
+        /// </summary>
+        public new bool IsFullyClosed => base.IsFullyClosed && RemainingPryCooldown > 0;
```

Comment:
it doesnt make any sense, those methods should be overridden from base with `override` keyword

Reply:
Isn't this would make them breaking change ?